modules/sdk-test-multi-cs/sdk-test-multi-cs.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Looks like a rebase artifact, this file should be removed. We don't have any sources in this folder since the multi-column test was moved to Codegen.Tests.